Firestore/Source/Remote/FSTRemoteStore.mm

```diff
@@ -239,14 +239,14 @@ - (void)listenToTargetWithQueryData:(FSTQueryData *)queryData {
 
   if ([self shouldStartWatchStream]) {
     [self startWatchStream];
-  } else if ([self isNetworkEnabled] && [self.watchStream isOpen]) {
+  } else if ([self isNetworkEnabled] && _watchStream->IsOpen()) {
```

Comment:
I think you can drop this isNetworkEnabled check.  We used to need it on android / JS to avoid a null pointer exception, since we used to null out the streams.  It was probably never technically needed on iOS since you can call methods on nil.  

In any case, the other ports no longer have this, so I think it should go.